labeling/internal/gitlab.go

```diff
@@ -0,0 +1,18 @@
+package internal
+
+import (
+	"path"
+
+	"github.com/CircleCI-Public/circleci-config/labeling/codebase"
+	"github.com/CircleCI-Public/circleci-config/labeling/labels"
+)
+
+var GitlabWorkflowRules = []labels.Rule{
```

Comment:
Seems like we don't have a test for gitlab - could we add a unit test for it?

Reply:
Add more tests on both the labelling and generation sides